lib/tasks/importer.rake

```diff
@@ -1,20 +1,19 @@
-require_relative '../cms/hippo_import'
-
 namespace :cms do
   namespace :hippo do
-    desc 'Import Hippo XML data into database (options: FROM=folder_name TO=site_identifier DOCS=file_of_docs)'
-
-    task :import => :environment do
+    desc 'Import Hippo XML data into database \n(options: FROM=folder_name TO=site_identifier DOCS=file_of_docs TYPE=type)'
+    task import: :environment do
       from  = ENV['FROM']
       to    = ENV['TO']
       docs  = ENV['DOCS']
+      type  = ENV['type'] || 'base'
```

Comment:
I tend to prefer `ENV.fetch('type', 'base')` but I know that's personal preference.